synth_data_module/formatting_helpers.py

```diff
@@ -0,0 +1,151 @@
+                                'justification': 'left'}
+        morbidity_list.append(present_on_admission)
+
+    return morbidity_list
+
+
+def modify_row(row, df_fields, new_fields):
+    first = row[df_fields[0]]
+    second = row[df_fields[1]]
+    if True:  # 'Procedure Codes' in df_fields:
+        if isinstance(first, tuple) and isinstance(second, tuple):
+            for code, date, i in zip(first, second, range(1, 26)):
+                if i == 1:
+                    if new_fields[0] == 'Diagnosis':
+                        row[f'Principal {new_fields[0]}'] = code
+                        row[f'{new_fields[1]} for Principal {new_fields[0]}'] = date
+                    else:
+                        row[f'Principal {new_fields[0]}'] = code
+                        row[f'Principal {new_fields[1]}'] = date
+                else:
+                    row[f'{new_fields[0]} {i}'] = code
+                    row[f'{new_fields[1]} {i}'] = date
+    return row
+
+# HCAI has specific criteria for age reporting
+def calculate_age(x, y, form="agedays"):
+    try:
+        date1 = pd.to_datetime(x, format='%m%d%Y')
+        date2 = pd.to_datetime(y, format='%m%d%Y')
+        agedays = date2-date1
+        age = relativedelta(date2, date1)
+    except:
+        return " "
+    # Days are only reported as a field if the patient is under 1 years old, else reported as "0"
+    if form == "agedays":
+        if agedays.days < 366:
+            return max(agedays.days, 1)
+        else:
+            return 0
+    if form == "staydays":
+        return agedays.days
+    elif form == 'adjstaydays':
+        return max(agedays.days, 1)
+    elif form == 'years':
+        return age.years
+    elif form == 'range5':
+        return str(math.ceil((age.years+1)/5)+math.ceil(1*(age.years/(age.years+1)))).zfill(2)
+    elif form == 'range10':
+        return str(math.ceil((age.years+1)/10)+math.ceil(1*(age.years/(age.years+1)))).zfill(2)
+
+
+# Functions for maintaining data outputs and processing arguments for runtime
+def clear_old_files():
+    output_cpcds = 'output/cpcds/*'
+    output_csvs = 'output/csv/*'
+    output_fhirs = 'output/fhir/*'
+    output_metadata = 'output/metadata/*'
+
+    files0 = glob.glob(output_cpcds)
+    for f in files0:
+        os.remove(f)
+    files1 = glob.glob(output_csvs)
+    for f in files1:
+        os.remove(f)
+    files2 = glob.glob(output_fhirs)
+    for f in files2:
+        os.remove(f)
+    files3 = glob.glob(output_metadata)
+    for f in files3:
+        os.remove(f)
+
+
+def parse_city(specify_city=None):
+    city = None
+    state = None
+    if specify_city:
+        if len(specify_city.split(",")) == 2:
+            city = str(specify_city.split(",")[0])
+            state = str(specify_city.split(",")[1])
+            print("Overriding default location Massachusetts to city, state: %s, %s " % (city, state))
```

Comment:
There are a lot of places where you have semi-info, semi-debug statements like this. They're fine for development, but if you want to leave them in you probably want to use the [`logging`](https://docs.python.org/3/library/logging.html) module more extensively. This will make it easier to turn these print statements on or off without going through all of the code.